src/PackagePrivate/Humanizer/InternationalizedHumanizer.php

```diff
@@ -153,7 +156,7 @@ private function humanizeYear( int $year, UnspecifiedDigit $unspecifiedDigit ):
     {
 	    $endingChar = $this->needsYearEndingChar($unspecifiedDigit) ? 's' : '';
 
-		return $year >= 0 ? (string)$year . $endingChar : (string)(-$year) . ' BC';
+		return $year >= 0 ? (string)$year . $endingChar : (string)(-$year) . $this->message('edtf-bc');
```

Comment:
This works for now, but beware that `$this->message('edtf-bc', (string)(-$year));` would probably be better, since then the suffix assumption is not hardcoded here.